LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
You should keep the license file. I'll merge this change, but please put it back in next commit.